src/bson/ext/int.cr

```diff
@@ -1,3 +1,27 @@
+struct Int8
+  # Serialize to a canonical extended json representation.
+  #
+  # NOTE: see https://github.com/mongodb/specifications/blob/master/source/extended-json.rst
+  def to_canonical_extjson(builder : JSON::Builder)
+    builder.object {
+      builder.string("$numberInt")
+      builder.scalar(Int32.new(self).to_s)
+    }
+  end
+end
+
+struct Int16
```

Comment:
Same here

Reply:
Fixed in following commit : 988b1c24c7e520fec575478e0ba307c40b779a3e